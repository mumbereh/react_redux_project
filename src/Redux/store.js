import { configureStore } from '@reduxjs/toolkit';
import homepageReducer from './Home/HomePageSlice';

const store = configureStore({
  reducer: {
    homepage: homepageReducer,
  },
});

export default store;
