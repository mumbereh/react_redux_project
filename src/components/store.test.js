import store from '../Redux/store';

describe('Store', () => {
  test('store matches the snapshot', () => {
    expect(store.getState()).toMatchSnapshot();
  });
});
