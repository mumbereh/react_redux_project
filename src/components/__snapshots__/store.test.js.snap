// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Store store matches the snapshot 1`] = `
Object {
  "homepage": Object {
    "categories": Array [],
    "error": null,
    "filter": "",
    "loading": false,
  },
}
`;
